Managed Services/what-happens-when-RedHat-servers-are-made-managed.md

```diff
@@ -0,0 +1,50 @@
+{{{
+  "title": "What Happens When RedHat Servers Are Made Managed",
+  "date": "07-29-2015",
+  "author": "Ben Swoboda",
+  "attachments": [],
+  "contentIsHTML": false
+}}}
+
+## Overview
```

Comment:
Headers should start at `###`. Please correct all the headers accordingly.

https://github.com/centurylinkcloud/publickb#headers